modules/functions.js

```diff
@@ -0,0 +1,92 @@
+export const taskEntry = document.getElementById('txtTask');
+export const tasks = [];
+const setStorage = () => {
+  localStorage.setItem('taskData', JSON.stringify(tasks));
+};
+const taskPopulate = (tasks) => {
+  const taskList = document.getElementById('container');
+  taskList.innerHTML = tasks.map((task) => `
+    <div class="task ${task.completed ? 'completed' : ''}">
+    <input type="checkbox" ${task.completed ? 'checked' : ''}id="${task.index}">
+    <label for = "${task.index}"> "${task.description}"</label>
+    <div class="remove"><button class="btnremove"><i class="fa">&#xf014;</i></button></div>
+    <div class="remove"><button class="btnEdit"><i class="fa-solid fa-pen-to-square"></i></button></div>
+
```

Comment:
- [ ] Kindly work on the display of elements, currently, all the elements despair from the screen when a user refreshes the browser. I have attached an image for your reference.

![reflesh](https://user-images.githubusercontent.com/53124548/236433819-61457917-bd7e-45cd-a104-b80e1b13f06b.png)